.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI/PR
+on: [push, pull_request]
```

Comment:
I copied this from git.git's ci setup. It's nice to kick off the build immediately on push, so that it's often finished by the time you've opened up the PR. But in git.git, we have some logic to avoid running the tests twice for both the push and the PR. We could probably use something like https://github.com/fkirc/skip-duplicate-actions/#skip-duplicate-workflow-runs here. But maybe we should just stick to running on pull requests; once the setup-ruby bits are cached, it's really pretty fast to run the whole CI process.